types/youtube/index.d.ts

```diff
@@ -697,6 +697,16 @@ declare namespace YT
       yaw?: number | undefined;
     }
 
+    export class get extends Player {
+      /**
+       * Get the instance of the Player class attached to an iframe.
+       *
+       * @param iframeId   Id of the iframe.
+       * @param options   Player options.
+       */
+      constructor(iframeId: string);
+    }
+
```

Comment:
I don't think `get` is a `class`. It looks like it's just a function that returns a registered player by its `iframe`'s id, so you shouldn't need to call `new`.

```suggestion
   /**
    * Get the instance of the Player class attached to an iframe.
    *
    * @param iframeId   Id of the iframe.
    */
    export function get(iframeId: string): Player | undefined;

```

Reply:
I made the change.

On my side, I always use it with a new and never got any error of this type
<img width="490" alt="image" src="https://user-images.githubusercontent.com/20707343/184101358-b9515715-4dad-4bd4-aeaf-3a924a4f8b85.png">

this is what I get when doing a new
<img width="376" alt="image" src="https://user-images.githubusercontent.com/20707343/184101401-28773f5e-2f73-4870-9a1b-4c2dc27db1e0.png">

Reply:
I just skimmed the YT iframe reference [here](https://developers.google.com/youtube/iframe_api_reference?hl=en) to see if I could figure this one out, but I don't see this function documented at all. Do you have a reference that shows this API?

The PR template asks for this, but you delete the template entirely.